client/src/views/Movie/Movie.view.jsx

```diff
@@ -1,26 +1,53 @@
 import { useState, useEffect } from "react";
-import { useParams } from "react-router-dom";
+import { useParams, Link } from "react-router-dom";
 
-import { MoviesList } from "components";
+import {
+  Header,
+  Main,
+  ErrorBoundary,
+  MovieDetail,
+  MainHeader,
+} from "components";
+
+import { ReactComponent as Search } from "icons/search.svg";
+
+import { convertMinsToHrsMins } from "converters";
 
 import { fetchMovieDetails } from "services/movieshelf-api";
 import Status from "utils/state-machine";
 const { IDLE, PENDING, REJECTED, RESOLVED } = Status;
 
 export const Movie = () => {
   const { movieId } = useParams();
-
   const [movie, setMovie] = useState(null);
   const [status, setStatus] = useState(IDLE);
   const [error, setError] = useState(null);
 
   useEffect(() => {
     setStatus(PENDING);
     fetchMovieDetails(movieId)
-      .then((movie) => {
-        setMovie(movie);
-        setStatus(RESOLVED);
-      })
+      .then(
+        ({
+          title,
+          vote_average: voteAverage,
+          genres,
+          release_date: releaseDate,
+          runtime,
+          poster_path: posterPath,
+          overview,
+        }) => {
+          setMovie({
+            title,
+            voteAverage,
+            genres,
+            releaseYear: new Date(releaseDate).getFullYear(),
+            runtime: convertMinsToHrsMins(runtime),
+            posterPath,
+            overview,
+          });
```

Comment:
it's better to order shorthand properties at the beginning, same on 31-37 lines
https://github.com/airbnb/javascript#objects--grouped-shorthand